### PR TITLE
Author user relationship2

### PR DESCRIPTION
**Pull request by crowbar72**

### CODE REVIEW

File: App/tests/test_app.py [hunk 1]
```diff
@@ -69,32 +69,28 @@ def test_author_toJSON(self):
...
         coauthor = Author("Bob Dule", "06/09/2002", "BSc. Computer Engineering")
         coauthors.append(coauthor)
-        publication = Publication("Intro to Computer Science", authors, coauthors)
+        publication = Publication("Intro to Computer Science", author.id, coauthors)
         publication_json = publication.toJSON()
         self.assertDictEqual(publication_json, {
             "id": None,
             "title": "Intro to Computer Science",
-            "authors": [author.toJSON() for author in authors],
+            "author": None,
```
Comment (isaac-cabrices) [inline App/tests/test_app.py:93]: Would this return the author.id as null? I think sqlalchemy auto-generates a primary key here.

Comment (crowbar72) [reply App/tests/test_app.py:93]: When I did the tests from the last project the id for unit tests was returned as null.

**Review by isaac-cabrices — APPROVED** [review]
Looks good!

File: App/tests/test_app.py (hunk 1 above)
Comment (isaac-cabrices) [reply App/tests/test_app.py:93]: Huh alright, I'll have to read up on that.

**Pull request merged**